javers-core/src/main/java/org/javers/guava/GuavaAddOns.java

```diff
@@ -0,0 +1,50 @@
+package org.javers.guava;
+
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Multiset;
+import org.javers.common.collections.Lists;
+import org.javers.common.reflection.ReflectionUtil;
+import org.javers.core.JaversBuilder;
+import org.javers.core.metamodel.object.GlobalIdFactory;
+import org.javers.core.metamodel.type.TypeMapper;
+import org.javers.guava.multimap.MultimapComparator;
+import org.javers.guava.multiset.MultisetComparator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * @author akrystian
+ */
+public class GuavaAddOns{
```

Comment:
Z tego co pamiętam to w beforeAssemble był problem, że `GlobalIdFactory` nie było zainicjowane.